Software/Development/SemesterProject/App/Include/digitalInputManager.h

```diff
@@ -17,7 +17,12 @@
 // Function prototyping
 //
 void readDigitalInputs(void);
+int isRegenBrakingPushbuttonEnabled(void);
```

Comment:
How about making all this functions boolean?

Reply:
Boolean is not a part of C as standard.